content/en/tracing/trace_pipeline/trace_retention.md

```diff
@@ -18,35 +18,35 @@ further_reading:
 
 {{< img src="tracing/apm_lifecycle/retention_filters.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Retention filters" >}}
 
-With APM, [the ingestion and the retention of traces for 15 days][1] are fully customizable.
+With Datadog APM, [the ingestion and the retention of traces for 15 days][1] are fully customizable.
 
 To track or monitor your volume of ingested and indexed data, see the [Usage Metrics][2] documentation.
 
 ## Retention filters
 
-After spans have been ingested by Datadog, some are kept for 15 days according to the retention filters that have been set on your account.
+After spans have been ingested, some are kept for 15 to 30 days according to the retention filters that have been set on your account.
 
-By default, some retention filters are enabled for your account:
-- The `Error Default` retention filter indexes error spans with `status:error`. The retention rate and the query can be configured. For instance, to capture production errors, set the query to `status:error, env:production`. Disable the retention filter if you do not want to capture the errors by default.
+Some retention filters are enabled by default, in order to ensure that you keep visibility over all of your services and endpoints, as well as errors and high-latency traces: 
+- The [Intelligent Retention Filter](#datadog-intelligent-retention-filter) retains spans for every environment, service, operation and resource for different latency distributions.
+- The `Error Default` retention filter indexes error spans with `status:error`. The retention rate and the query are configurable. For instance, to capture production errors, set the query to `status:error, env:production`. Disable the retention filter if you do not want to capture the errors by default.
 - The `Application Security` retention filter is enabled if you are using Application Security Management. It ensures the retention of all spans in traces that have been identified as having an application security impact (an attack attempt).
-- The [Intelligent Retention Filter](#datadog-intelligent-retention-filter) retains diverse error traces and traces from different latency distributions.
 
-You can also create any number of additional [tag-based retention filters](#create-your-own-retention-filter) for your services.
+On top of these, you can create any number of additional [custom tag-based retention filters](#create-your-own-retention-filter) for your services, to capture the data that matters the most to your business.
 
-**Note**: Admin rights are required to create, modify, or disable retention filters.
+**Note**: The permission `apm_retention_filter_write` is required to create, modify, or disable retention filters.
 
```

Comment:
permission `apm_retention_filter_write` is required to create, delete, modify, or enable/disable retention filters.